Validated.md

```diff
@@ -0,0 +1,363 @@
+Invalid("InvalidA").findValid(plusStringSemigroup, { Invalid("InvalidB") })
+// Invalid("InvalidA, InvalidB")
+```
+
+---
+
+# Validated :: Applicative Builder
+
+Λrrow contains methods that allow you to preserve type information when computing over different __Validated__ typed values.
+
+```kotlin
+object KnownError
+data class Person(id: UUID, name: String, year: Int)
+
+// Note each Validated is of a different type
+val vId: Validated<KnownError, UUID> = Valid(UUID.randomUUID())
+val vName: Validated<KnownError, String> = Valid("William Alvin Howard")
+val vAge: Validated<KnownError, Int> = Valid(1926)
+
+Validated.applicative<KnownError>().map(vId, vName, vAge, { (id, name, age) ->
+  Person(id, name, age)
+}).ev()
+// Valid(Person(<uuid>, "William Alvin Howard", 1926))
+```
+
+---
+
+# Validated :: Applicative Builder
+
+If a value turns out to be __Invalid__ the computation doesn't short-circuit, but instead combines the possible invalid values encountered.
+
+```kotlin, [.highlight: 6, 12]
+object KnownError
+data class Person(id: UUID, name: String, year: Int)
+
+// Note each Validated is of a different type
+val vId: Validated<KnownError, UUID> = Invalid(KnownError)
+val vName: Validated<KnownError, String> = Invalid(KnownError)
+val vAge: Validated<KnownError, Int> = Right(1926)
+
+Validated.applicative<KnownError>().map(vId, vName, vAge, { (id, name, age) ->
+  Person(id, name, age)
+}).ev()
+// Invalid(KnownError |+| KnownError)
+```
+
+---
+
+# Validated :: Conclusion
+
+- Validated is __used to model a value that we can consider valid or invalid__ under our context
+- We can easily construct values of `Validated` with `Invalid(1)`, `Valid(1)`, `1.invalid()` or `1.valid()`.
+- __fold__, __map__ and others are used to compute over the internal contents of a Validated computation.
+- __Validated.applicative<E>().map { ... }__ can be used to compute over multiple validated values preserving type information and __abstracting over arity__ with `map`
+
+---
+
+# Validated :: Conclusion
+
+- __All techniques demonstrated are also available to other data types__ such as `Either`, `Option`, `IO` and you can build adapters for any data types.
+- We will learn more about other data types like `Either`, `Option`, `IO` and type classes that power these abstraction such as `Functor`, `Applicative` and `Monad` in other videos.
+- __Λrrow encourages a unified programming model__ in which you can solve problems cohesively in all contexts following Typed Functional Programming principles applied to the Kotlin Programming Language.
+
+---
+
+Thanks for watching!
+
+- Λrrow : [http://arrow-kt.io](http://arrow-kt.io)
+- Gitter : [https://gitter.im/arrow-kt/Lobby](https://gitter.im/arrow-kt/Lobby)
+- Slack : [https://kotlinlang.slack.com/messages/C5UPMM0A0](https://kotlinlang.slack.com/messages/C5UPMM0A0)
+- FP with Arrow
+- 47 Degrees : [http://47deg.com](http://47deg.com)
+- @calvellido : [https://twitter.com/raulraja](https://twitter.com/calvellido)
```

Comment:
Looks like you still have raul's handle on the link 😄

Reply:
woopsie! 😅 Nice catch, thanks Justin!